File: Assignment5/program5.c
#include<stdio.h>
// TIME COMPLEXITY = O(1)   {O(5) IS CONSIDERED AS O(1)}
void MultipleDisplay(int iNo)
{
  int iCnt=0, iMultiple=0;
  for(iCnt=1;iCnt<=5;iCnt++)
  {
    iMultiple = iCnt * iNo;
    printf("%d ",iMultiple);
    iMultiple=0;
  }
 
}


int main()
{
  int iValue = 0;
  printf("Enter the number : ");
  scanf("%d",&iValue);
  MultipleDisplay(iValue);
  return 0;
}

File: Assignment11/program5.java
import java.util.Scanner;
// TIME COMPLEXITY O(N)
class Pattern
{
  public void Display(int iCol)
  {
    int i = 0;
    for(i=1 ;i<=(2*iCol);i++)
    {
      if((i % 2) == 0 )
      {
        System.out.print(i+"\t");
        
      }
    }
    System.out.println("");
  }
}
public class program5
{
  public static void main(String[] args)
   {
      int iValue = 0;
      Scanner sobj = new Scanner(System.in);
      System.out.println("Enter the number of Elements: ");
      iValue = sobj.nextInt();
      Pattern pobj = new Pattern();
      pobj.Display(iValue);

  }
}
